Sources/Widgets/Widgets3D/PolyLineWidget/behavior.js

```diff
@@ -37,7 +38,15 @@ export default function widgetBehavior(publicAPI, model) {
         model._isDragging) &&
       model.activeState.setOrigin // e.g. the line is pickable but not draggable
     ) {
-      model.activeState.setOrigin(worldCoords);
+      if (model._isDragging) {
+        // only use the drag helper if we are dragging a handle,
+        // not moving the move handle
```

Comment:
I think this documentation could go in `handleEvent`, to know when to rely on worldDelta, and when not to rely on worldDelta. 
It is not obvious why dragging a handle requires a drag helper (which is an obsolete name ?)

Reply:
This is an obsolete comment that needs to be removed.